rdt/hyper_transformer.py

```diff
@@ -73,6 +75,42 @@ class HyperTransformer:
         'b': 'boolean',
         'M': 'datetime',
     }
+    DEFAULT_TRANSFORMERS = {
+        'numerical': NumericalTransformer,
+        'integer': NumericalTransformer(dtype=int),
+        'float': NumericalTransformer(dtype=float),
+        'categorical': CategoricalTransformer(fuzzy=True),
+        'boolean': BooleanTransformer,
+        'datetime': DatetimeTransformer,
+    }
+    _DTYPES_TO_DATA_TYPES = {
+        'i': 'integer',
+        'f': 'float',
+        'O': 'categorical',
+        'b': 'boolean',
+        'M': 'datetime',
+    }
+
+    @staticmethod
+    def get_transformers_by_type():
```

Comment:
I think it would make more sense to put this as a function inside the `transformers` module, either directly in the `__init__` module or in a `utils` module and then importing it from the `__init__`.

I imagine the use case where someone simply wants to explore what transformers exist, without using the `HyperTransformer`

```
>>> import rdt
>>> rdt.transformers.get_transformers_by_type()
...
```